var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -265,12 +268,36 @@ def blas_lapack_pkg_config(self) -> Tuple[str, str]:
 
         return blas, lapack
 
-    @when("@1.26:")
+    @when("@1.25:")
     def config_settings(self, spec, prefix):
+        settings = {"builddir": "build", "compile-args": f"-j{make_jobs}"}
+
+        if self.spec.satisfies("@1.26:"):
+            settings.update(self.blas_config_settings())
+
+        # Disable AVX512 features for Intel Classic compilers
+        # https://numpy.org/doc/stable/reference/simd/build-options.html
+        # https://github.com/numpy/numpy/issues/27840
+        # https://github.com/matplotlib/matplotlib/issues/28762
+        archs = ("x86_64_v4:", "cannonlake:", "mic_knl")
+        if any([self.spec.satisfies(f"target={arch} %intel") for arch in archs]):
+            intel_settings = {
+                "setup-args": {
+                    "-Dcpu-dispatch": (
+                        settings.get("setup-args", {}).get("-Dcpu-dispatch", "")
+                        + " "
```

Comment:
This line can be combined with later lines

Reply:
Removed.